config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml

```diff
@@ -163,48 +163,14 @@ spec:
   heat:
     enabled: false
     template:
-      customServiceConfig: "# add your customization here"
       databaseInstance: openstack
-      databaseUser: "heat"
       rabbitMqClusterName: rabbitmq
-      debug:
-        bootstrap: false
-        dbSync: false
-        service: false
       heatAPI:
-        customServiceConfig: "# add your customization here"
-        databaseUser: "heat"
-        debug:
-          bootstrap: false
-          dbSync: false
-          service: false
-        passwordSelectors:
-          service: AdminPassword
-          database: AdminPassword
         replicas: 1
-        resources: {}
-        secret: "osp-secret"
-        serviceUser: ""
       heatEngine:
-        customServiceConfig: "# add your customization here"
-        databaseUser: "heat"
-        debug:
-          bootstrap: false
-          dbSync: false
-          service: false
-        passwordSelectors:
-          service: AdminPassword
-          database: AdminPassword
         replicas: 1
-        resources: {}
-        secret: "osp-secret"
-        serviceUser: ""
-      passwordSelectors:
-        service: AdminPassword
-        database: AdminPassword
-      preserveJobs: false
+      rabbitMqClusterName: rabbitmq
```

Comment:
I think this is the one pre-commit is complaining about.

Reply:
Yeah, easier to see the indentation issue in this view:
https://github.com/openstack-k8s-operators/openstack-operator/blob/aa26499c3d330c898c90b0e3223ccb8eb0d90c49/config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml#LL163C1-L173C25